docs/card-present-payments.md

```diff
@@ -0,0 +1,54 @@
+
+1. Obtaining a token
+2. Creating and processing a payment intent using the card reader
+3. Capturing the payment intent submitting it to WooCommerce Payments.
+
+### Obtaining a token
+
+The Stripe Terminal SDK requires a connection token as a prerequisite to any interaction with the SDK.
+
+The way to provide that token to the SDK is by providing an implementation of a protocol declared in the Stripe Terminal SDK, called `ConnectionTokenProvider`
+
+The mobile apps obtain the token from the endpoint `payments/connection_tokens`. The integration with this endpoint is implemented in WCPayRemote.
+
+### Creating and processing a payment intent using the card reader
+
+In order to obtain a payment intent, the mobile app needs to discover and connect to a card reader first.
+
+Once the discovery and connection process is completed, we can collect a payment. To do so, we rely on the Stripe Terminal SDK, which will execute three operations in sequence: 
+1. Create a payment  intent. This is the initial step of the process. The Stripe Terminal SDK transitions into a mode where it is ready to start collecting a payment. To do so, it creates a payment intent, which will be getting updated as the payment collection process advances.
+2. Collect a payment method. We collect a payment method by asking users to tap/insert/swipe their cards. 
+3. Process the payment. This is a somewhat obscure operation, where we assume the Terminal SDK submits all the information it has collected so far to Stripe. This information will have to be matched with what the mobile app provides in the next phase.
+
+### Capturing a payment
+
+Once we have a payment intent that has been processed by the Stripe Terminal SDK, we submit the payment intent identifier to an endpoint in WooCommerce Payments (`payments/orders/{id}/capture_terminal_payment`). WooCommerce Payments will then proxy the request containing this identifier to Stripe, where all the pertinent checks will be performed, and the response will be sent back down to the mobile app. 
+
+The end result is a "OK" or "KO" that the mobile app then uses to present a success or error message to the user.
+
+### Printing a receipt
+
+If everything goes well in the previous step, and WooCommerce Payments lets us know that the payment intent identifier was captured correctly, and the payment was finally captured, it is time for us to offer merchants the option to print or email a receipt.
+
+There is a caveat here: when we initiate the process to capture a payment, we will attempt to provide the customer email address to the Stripe Terminal SDK. If the request to create a payment intent contains a valid email address, Stripe will send a receipt automatically.
+
+Leaving that aside, we still offer the option to print or email a receipt right after the payment has been collected and processed. 
+
+Orders that have been processed via a Card Present Payment will also offer the option to see a receipt after the fact. 
```

Comment:
Do we want to mention that this is implemented locally for now? For instance, to make clear that receipts won't survive a reinstall, or be available across devices.